### DISCUSSION TITLE
[FEATURE] Disclaimer?

### DISCUSSION THREAD

**Discussion by JulieBB23** [gc-da11yn/Content-and-ux#9] [Ideas]
### Is your feature request related to a problem? Please describe.

Should we have a disclaimer like W3C has?
here the content:
**Important Disclaimer**
W3C does not endorse specific vendor products. Inclusion of products in this list does not indicate endorsement by W3C. Products and search criteria are listed with no quality rating.

Tool descriptions, search criteria, and other information in this database is provided by tool developers, vendors, or others. W3C does not verify the accuracy of the information.

The list is not a review of evaluation tools, nor a complete or definitive list of all tools. The information can change at any time.

[https://www.w3.org/WAI/ER/tools/](https://www.w3.org/WAI/ER/tools/)
here an example of what W3C has for the tools page:
Important Disclaimer
W3C does not endorse specific vendor products. Inclusion of products in this list does not indicate endorsement by W3C. Products and search criteria are listed with no quality rating.

Tool descriptions, search criteria, and other information in this database is provided by tool developers, vendors, or others. W3C does not verify the accuracy of the information.

The list is not a review of evaluation tools, nor a complete or definitive list of all tools. The information can change at any time.

### Who shall do the work?

- [X] I will do it, I want to contribute
- [X] I am asking for someone else to please do the work

**Comment by shawnthompson** [comment]
👍  but where do we put this disclaimer?

**Comment by shawnthompson** [comment]
Maybe we can add it to a [Contextual band and sub-footer band - Canada.ca](https://wet-boew.github.io/GCWeb/sites/footers/no-footer-main-en.html) section.

Maybe we can add a link to the [page that lists all our pages](https://a11y.canada.ca/en/page-list/) in the footer too.

**Comment by shawnthompson** [comment]
Related to: https://github.com/gc-da11yn/gc-da11yn.github.io/issues/328
